pyccel/codegen/printing/ccode.py

```diff
@@ -1253,7 +1254,6 @@ def _print_NumpyUfuncBase(self, expr):
             numpy.cos(x) ==> cos(x)
 
         """
-        # add necessary include
         self.add_import(c_imports['math'])
```

Comment:
Unnecessary removal of comment

Reply:
https://github.com/pyccel/pyccel/pull/1169/commits/145ea58b2db75e69eb450efe233f7f1def24ca2f